apps/simple-trading-app/src/app/components/deal-ticket/deal-ticket-container.tsx

```diff
@@ -4,15 +4,52 @@ import {
   DealTicketContainer as Container,
 } from '@vegaprotocol/deal-ticket';
 import { DealTicketSteps } from './deal-ticket-steps';
+import { useVegaWallet } from '@vegaprotocol/wallet';
+import { gql, useQuery } from '@apollo/client';
+import { DealTicketBalance } from './deal-ticket-balance';
+import * as React from 'react';
 
 const tempEmptyText = <p>Please select a market from the markets page</p>;
 
+const PARTY_BALANCE_QUERY = gql`
+  query PartyBalanceQuery($partyId: ID!) {
+    party(id: $partyId) {
+      accounts {
+        balance
+        asset {
+          id
+          symbol
+          name
+          decimals
+        }
+      }
+    }
+  }
+`;
+
 export const DealTicketContainer = () => {
   const { marketId } = useParams<{ marketId: string }>();
+  const { keypair } = useVegaWallet();
+
+  const { data: partyData, loading } = useQuery(PARTY_BALANCE_QUERY, {
```

Comment:
suggestion (non blocking): you should be able to run nx run types:generate to generate types for this query which you can pass in as type arguments to useQuery

Reply:
I did run it, might have forgot to add it. Will refactor :)